tracks/csharp/exercises/space-age/mentoring.md

```diff
@@ -0,0 +1,84 @@
+### Reasonable solution
+
+- using Dictionary
+    ```csharp
+    using System.Collections.Generic;
+
+    public class SpaceAge
+    {
+        private readonly double earthYear;
+
+        private static Dictionary<string, double> conversionFactor 
```

Comment:
Instead of using a string as the key, maybe we should be using an enumeration here?

Reply:
I would also include the conversion factor to earth here. I include, the `earthYear / conversionFactor[<planet>]` code could be extracted to a helper method.

Reply:
I will apply the helper method

Reply:
My only comment would be related to the enum within the solution with _Dictionary_ . While enum is perfect for a small set of values, using Dictionary potentially opens the solution for a more general implementation (when we could have thousands of planets from the sky, and generally compare them with Earth).